exonum/src/api/node/public/system.rs

```diff
@@ -25,20 +25,31 @@ pub struct MemPoolInfo {
     pub size: u64,
 }
 
-/// Information about the amount of peers connected to the node.
-#[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
-pub struct PeersAmount {
-    /// Amount of connected peers.
-    pub amount: usize,
-}
-
 /// Information about whether the node is connected to other peers.
 #[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
-pub enum ConnectivityStatus {
+pub struct ConnectivityStatus {
     /// The node has no connected peers.
-    NotConnected,
+    connected: bool,
```

Comment:
Is the flag needed if we include the number of connections (peers=0 -> not connected)?

Reply:
@dmitry-timofeev I asked you and @bullet-tooth in the ticket. If you see in the ticket so the answer was "Format is ok"

Reply:
But that doesn't answer if including a boolean makes sense. Is it possible to have peers != 0 but connected = false? If connected always implies > 0 connections and 0 connections imply not connected, then it's just redundant data.

Reply:
Basically, you're right. If connections number is 0 then flag is always `false`. Provide a format of response in the ticket please.

Reply:
I see, thanks! I'll get back to that shortly (maybe @bullet-tooth would be able to clarify that first?)

Reply:
Agree with @dmitry-timofeev for removing this flag. Ticket updated with new format